hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/receiver/RetryableReceiverError.java

```diff
@@ -0,0 +1,9 @@
+package pl.allegro.tech.hermes.consumers.consumer.receiver;
+
+import pl.allegro.tech.hermes.common.exception.InternalProcessingException;
+
+public class RetryableReceiverError extends InternalProcessingException {
```

Comment:
I decided to add some exception which is understandable by higher level of abstraction to avoid making for layer.